vsts/internal-version-increment.sh

```diff
@@ -75,3 +75,8 @@ fi
 if [ "$BUILD_SOURCEBRANCHNAME" != "master" ]; then
   sed -i '' 's/\(VERSION_STRING[[:space:]]*=[[:space:]]*\).*/\1'$publish_version'/g' $VERSION_FILENAME
 fi
+sed '/define("APP_CENTER_C_VERSION"*/c\'$'\n''dd'$'\n''' Package.swift
+package_version_text='"\"'$publish_version'\""'
+latest_build_number_text='"\"'$latest_build_number'\""'
+sed 's/\(define("APP_CENTER_C_VERSION",[[:space:]]*to:*\).*/\1'$package_version_text'/g' Package.swift
```

Comment:
I have two issues with these sed commands locally.
- Package.manifest is not updated, I see only console output,
- wrong format, no closing bracket:
` .define("APP_CENTER_C_BUILD", to:500523`